keys-controller.js

```diff
@@ -88,6 +95,15 @@ function factory($scope, $routeParams, brAlertService, brKeyService,
   });
 }
 
+// TODO: allow users to modify specific resources
+function hasPermission(identity, permission) {
+  // if sysPermissionTable is an object, it is a map of specific resources
+  // to which this permission applies.
+  // boolean value indicates that the user has unrestricted permissions
+  return (permission in identity.sysPermissionTable &&
+    typeof identity.sysPermissionTable[permission] === 'boolean');
```

Comment:
`if(sysPermissionTable[permission] === true)` will not resolve `true` if sysPermission is an object, it is using triple equals (which will check the type -- and which should always be used).

https://repl.it/CI9s